tmt/base.py

```diff
@@ -456,6 +456,12 @@ def export(self, format_='yaml', keys=None):
             tmt.export.export_to_nitrate(self)
 
         # Common node export otherwise
+        elif keys == 'fmf-id':
+            if format_ == 'dict':
```

Comment:
Do we need dict? If yes please mention it in help.

Reply:
Not sure, "tmt tests export" supports "--format dict" so I thought it might be good for consistency. Let me know if we do not need it.

Reply:
Added explicit mention to the help.